Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -582,20 +584,16 @@ std::pair<std::string, std::shared_ptr<HoI4::Country>> HoI4::World::getDominion(
 	 Names& names,
 	 Localisation& hoi4Localisations)
 {
-	if (const auto& dominionTag = dominions.find(std::make_pair(ownerTag, region)); dominionTag != dominions.end())
+	if (dominions.contains(std::make_pair(ownerTag, region)))
```

Comment:
I prefer the old-style check here. It saves you the second pass through dominions for the at(), as you already have the dominion from the lookup.